docs/data/material/pages.ts

```diff
@@ -310,6 +310,22 @@ const pages: MuiPage[] = [
       },
     ],
   },
+  {
+    pathname: '/material-ui/design-resources',
```

Comment:
I think this should come after "/material-ui/discover-more". It looks like the direction is move toward having all of these docs outside of https://mui.com/material-ui/ because it's:

- not open source so can't be mixed with open-source Material UI, we can't create confusion for what's free and paid.
- part of it is for designers (the Figma kit)
- won't be just about Material UI, but also Base UI, MUI X